Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -159,6 +159,35 @@ def setup_remote_tap(name)
     tap
   end
 
+  def install_and_rename_coretap_formula(old_name, new_name)
+    core_tap = CoreTap.new
+    core_tap.path.cd do
+      shutup do
+        system "git", "init"
+        system "git", "add", "--all"
+        system "git", "commit", "-m",
+          "#{old_name.capitalize} has not yet been renamed"
+      end
+    end
+
+    cmd("install", old_name)
+    (core_tap.path/"Formula/#{old_name}.rb").unlink
+    formula_renames = core_tap.path/"formula_renames.json"
+    formula_renames.write <<-EOS.undent
+    {
+      "#{old_name}\": \"#{new_name}"
+    }
+    EOS
```

Comment:
The lines between `<<-EOS.undent` and `EOS` should be indented by two spaces. The escaped double quotation signs also look a bit wrong here. However, even better than constructing JSON data by hand (where it's easy to make a mistake) would be to just use:

```ruby
formula_renames.write Utils::JSON.dump(old_name => new_name)
```